src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/model/ViolationTicketCount.java

```diff
@@ -61,11 +61,40 @@ public class ViolationTicketCount {
     private String actOrRegulation;
 	
 	/**
-	 * The section of the act or regulation. For example, "147(1)" which means "Speed in school zone"
+	 * The full section of the act or regulation represented in a single string. For example, "147(1).A.MVA" which means "Speed in school zone"
 	 */
 	@Column
+	@Schema(nullable = true)
+    private String fullSection;
+	
+	/**
+	 * The section part of the full section. For example, "147"
+	 */
+	@Column
+	@Schema(nullable = true)
     private String section;
 	
+	/**
+	 * The subsection part of the full section. For example, "(1)"
+	 */
+	@Column
+	@Schema(nullable = true)
+    private String subsection;
+	
+	/**
+	 * The paragraph part of the full section. For example, "A"
+	 */
+	@Column
+	@Schema(nullable = true)
+    private String paragraph;
```

Comment:
Yeah, not sure if we should be splitting out the section/subsection/paragraph from the full Section.
What about the roman numerals often added, ie. (iv)?  The MVA Section data we have from JUSTIN often contains 4 groups, not 3.
24.1(1)(b.1)(viii)
24(4) Section (3)(a)
26(1)(b.1)(x)
170(b)(iii)

Also, I don't know if we should be data fixing JUSTIN data. I kinda think we should just reference and use their data, not try to parse and validate it into separate fields.